source/hands_on/gail.rst

```diff
@@ -80,6 +85,40 @@ Extensions
    implicit connection to GAN, IRL, and energy-based probability
    estimation.
 
+Implementation
+---------------------------------
+The default config is defined as follows:
+
+.. autoclass:: ding.reward_model.gail_irl_model.GailRewardModel
+
+Benchmark
+-----------
+
+
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+| environment         |best mean reward | evaluation results                                  | config link              | expert               |
++=====================+=================+=====================================================+==========================+======================+
+|                     |                 |                                                     |`config_link_l <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/| DQN                  |
+|Lunarlander          |  200            |.. image:: images/benchmark/lunarlander_gail.png     |box2d/lunarlander/config/ |                      |
+|                     |                 |                                                     |lunarlander_dqn_gail_     |                      |
+|(LunarLander-v2)     |                 |                                                     |config.py>`_              |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_b <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Bipedalwalker        |                 |                                                     |DI-engine/tree/main/dizoo/| SAC                  |
+|                     |  300            |.. image:: images/benchmark/bipedalwalker_gail.png   |box2d/bipedalwalker/      |                      |
+|(BipedalWalker-v3)   |                 |                                                     |config/bipedalwalker_sac_ |                      |
```

Comment:
line 112 and line 110 spelling of BipedalWalker is different

Reply:
changed both to double capital letter version